.travis/hbase-install.sh

```diff
@@ -1,6 +1,7 @@
 #! /bin/bash
 
-if [ ! -f $HOME/downloads/hbase-2.1.4-bin.tar.gz ]; then sudo wget -O $HOME/downloads/hbase-2.1.4-bin.tar.gz http://www-us.apache.org/dist/hbase/2.1.4/hbase-2.1.4-bin.tar.gz; fi
-sudo mv $HOME/downloads/hbase-2.1.4-bin.tar.gz hbase-2.1.4-bin.tar.gz && tar xzf hbase-2.1.4-bin.tar.gz
-sudo rm -f hbase-2.1.4/conf/hbase-site.xml && sudo mv .travis/hbase/hbase-site.xml hbase-2.1.4/conf
-sudo hbase-2.1.4/bin/start-hbase.sh
+if [ ! -f $HOME/downloads/hbase-2.1.5-bin.tar.gz ]; then sudo wget -O $HOME/downloads/hbase-2.1.5-bin.tar.gz http://www-us.apache.org/dist/hbase/2.1.5/hbase-2.1.5-bin.tar.gz; fi
```

Comment:
We need to bump HBase version in tests; the old version was deleted again from their server :/